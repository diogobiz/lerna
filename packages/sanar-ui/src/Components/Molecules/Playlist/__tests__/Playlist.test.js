
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESPlaylist from '../'

it('renders correctly', () => {
    const component = (
        <ESPlaylist>
            <h1>ESPlaylist</h1>
        </ESPlaylist>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
