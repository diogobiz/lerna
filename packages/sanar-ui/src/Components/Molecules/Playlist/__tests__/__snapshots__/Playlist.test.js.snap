// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="ant-skeleton es-skeleton es-playlist__description-skeleton es-skeleton--dark ant-skeleton-with-avatar ant-skeleton-active"
  >
    <div
      className="ant-skeleton-header"
    >
      <span
        className="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
        style={Object {}}
      />
    </div>
    <div
      className="ant-skeleton-content"
    >
      <ul
        className="ant-skeleton-paragraph"
      >
        <li
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <li
          style={
            Object {
              "width": "61%",
            }
          }
        />
      </ul>
    </div>
  </div>,
  <div
    className="ant-skeleton es-skeleton pl-md pr-md es-skeleton--dark ant-skeleton-active"
  >
    <div
      className="ant-skeleton-content"
    >
      <ul
        className="ant-skeleton-paragraph"
      >
        <li
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <li
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <li
          style={
            Object {
              "width": "100%",
            }
          }
        />
      </ul>
    </div>
  </div>,
]
`;
